fov_processing_pipeline/bin/process.py

```diff
@@ -51,57 +69,140 @@ def main():
         "--overwrite", type=utils.str2bool, default=False, help="overwite saved results"
     )
     p.add_argument(
-        "--use_current_results", type=utils.str2bool, default=False, help="dont do any processing. just make figures."
+        "--use_current_results",
+        type=utils.str2bool,
+        default=False,
+        help="dont do any processing. just make figures.",
+    )
+    p.add_argument(
+        "--debug",
+        type=utils.str2bool,
+        default=False,
+        help="Do debugging things (currently applies only to distributed)",
+    )
+    # distributed stuff
+    p.add_argument(
+        "--distributed",
+        type=utils.str2bool,
+        default=False,
+        help="Use Prefect/Dask to do distributed compute.",
+    )
+    p.add_argument(
+        "--port",
+        type=int,
+        default=99999,
```

Comment:
A "safer" method may be to do a random int between 10000 - 30000 or something :woman_shrugging: but this works!